derivation_method/action.py

```diff
@@ -1103,6 +1103,8 @@ def apply(
         api_url = os.environ.get("CLD_API_HOST")
         api_auth = os.getenv('CLD_API_AUTH')
 
+        api_url = api_url+"items/" if api_url.endswith('/') else api_url+"/items/"
```

Comment:
@PranjaliParnerkar 
Can you make it also backward-compatible? i.e. support all 4 options of env variable:
http://10.41.81.144:8005
http://10.41.81.144:8005/
http://10.41.81.144:8005/items
http://10.41.81.144:8005/items/